### PR TITLE
feat(manager) : Fixing containerized deployment reviews and adding port to the section

### PR DESCRIPTION
**Pull request by nikhitamore**
## Explain the feature/fix

1. The healthcheck path and the path field is not having / set as default
2. The health-check path is not taking values from the path attribute
3. When i am going inside spadetail, the name is coming as the property name. whether that was expected? ( the deployment is not happening for the spa which is selected)
4. Added port in the SSR form

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx [hunk 1]
```diff
@@ -81,11 +79,19 @@ export const ConfigureSSRForm = ({
...
+    getValues,
     formState: { isSubmitting }
   } = useForm<FormData>({
     mode: 'onBlur',
     resolver: yupResolver(schema),
-    defaultValues: { ...dataprops, config: keyValuePairsGenerator({ dataprops }) }
+    defaultValues: {
+      ...dataprops,
+      config: keyValuePairsGenerator({ dataprops }),
+      port: dataprops.port ? dataprops.port : '3000',
```
Comment (riginoommen) [inline packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx:91]:
```suggestion
      port: dataProps.port ? dataProps.port : '3000',
```

File: packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx [hunk 2]
```diff
@@ -81,11 +79,19 @@ export const ConfigureSSRForm = ({
...
+      path: dataprops.path ? dataprops.path : '/',
+      healthCheckPath: dataprops.healthCheckPath ? dataprops.healthCheckPath : '/'
```
Comment (riginoommen) [inline packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx:93]:
```suggestion
      healthCheckPath: dataProps.healthCheckPath ? dataProps.healthCheckPath : '/'
```

File: packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx [hunk 3]
```diff
@@ -81,11 +79,19 @@ export const ConfigureSSRForm = ({
...
     handleSubmit,
+    setValue,
+    getValues,
     formState: { isSubmitting }
   } = useForm<FormData>({
     mode: 'onBlur',
     resolver: yupResolver(schema),
-    defaultValues: { ...dataprops, config: keyValuePairsGenerator({ dataprops }) }
+    defaultValues: {
+      ...dataprops,
```
Comment (riginoommen) [inline packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx:89]:
```suggestion
      ...dataProps,
```

File: packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx (hunk 1 above)
Comment (nikhitamore) [reply packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx]: Done

File: packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx [hunk 4]
```diff
@@ -81,11 +79,19 @@ export const ConfigureSSRForm = ({
...
+      config: keyValuePairsGenerator({ dataprops }),
+      port: dataprops.port ? dataprops.port : '3000',
+      path: dataprops.path ? dataprops.path : '/',
```
[the remark this answers is not in the document]
Comment (nikhitamore) [reply packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx]: done

File: packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx (hunk 2 above)
Comment (nikhitamore) [reply packages/manager/src/views/WebPropertyDetailPage/components/SSR/ConfigureSSRForm.tsx]: done

**Review by soumyadip007 — APPROVED** [review]

**Review by riginoommen — APPROVED** [review]

**Pull request merged**